dev/reference/key-monitoring-metrics/tidb-dashboard.md

```diff
@@ -18,31 +18,60 @@ To understand the key metrics displayed on the TiDB dashboard, check the followi
 
 - Query Summary
     - Duration: the execution time of a SQL statement
+    - QPS: the statistics of OKs and Errors according to the SQL execution result on each TiDB instance
     - Statement OPS: the statistics of executed SQL statements (including `SELECT`, `INSERT`, `UPDATE` and so on)
     - QPS By Instance: the QPS on each TiDB instance
+    - Failed Query OPM: the statistics of error types (such as syntax errors and primary key conflicts) according to the errors happening when executing SQL statements on each TiDB instance
+    - Slow query: the statistics of the processing time of slow queries (the time cost of the entire slow query, the time cost of Coprocessor，and the waiting time for Coprocessor scheduling)
+    - 999/99/95/80 Duration: the statistics of the execution time for different types of SQL statements (different percentiles)
 
 - Query Detail
+    - Duration 80/95/99/999 By Instance: the statistics of the execution time for SQL statements on each TiDB instance (different percentiles)
+    - Failed Query OPM Detail: the statistics of error types (such as syntax errors and primary key conflicts) according to the errors happening when executing SQL statements in the entire cluster
     - Internal SQL OPS: the statistics of the executed SQL statements within TiDB
 
 - Server
+    - Uptime: the runtime of TiDB
+    - Memory Usage: the statistics of memory usage of different TiDB instances
+    - CPU Usage: the statistics of CPU usage of different TiDB instances
     - Connection Count: the number of clients connected to each TiDB instance
-    - Failed Query OPM: the statistics of failed SQL statements, such as grammar mistake, primary key, and so on
-    - Heap Memory Usage: the heap memory size used by each TiDB instance
+    - Open FD Count: the statistics of opened file descriptors of different TiDB instances
+    - Goroutine Count: the number of Goroutines of different TiDB instances
+    - Go GC Duration: the statistics of GC time of different TiDB instances
+    - Go Threads: the number of threads of different TiDB instances
+    - Go GC Count: the number of times that GC is executed on different TiDB instances
+    - Go GC CPU Usage: the statistics of GC CPU usage of different TiDB instances
     - Events OPM: the statistics of key events, such as "start", "close", "graceful-shutdown","kill", "hang", and so on
+    - Keep Alive OPM: the number of times that the monitor refreshes every minute on different TiDB instances
+    - Prepare Statement Count: the number of `Prepare` statements that are executed on each TiDB instance and the total count of them
+    - Time Jump Back OPS: the number of times that the time jumps back every second on different TiDB instances
```

Comment:
`jumps back => rewind`? (not sure which one is better)

Reply:
"rewind" is better. Thanks for pointing it out.